plugin/trino-mysql/src/main/java/io/trino/plugin/mysql/MySqlClient.java

```diff
@@ -316,6 +320,14 @@ protected String getTableSchemaName(ResultSet resultSet)
         return legacyToPrestoType(session, connection, typeHandle);
     }
 
+    private static Optional<ColumnMapping> getVarcharType(int varcharLength)
+    {
+        VarcharType varcharType = (varcharLength <= VarcharType.MAX_LENGTH) ? createVarcharType(varcharLength) : createUnboundedVarcharType();
+        // Remote database can be case insensitive.
+        PredicatePushdownController predicatePushdownController = CASE_INSENSITIVE_PUSHDOWN;
```

Comment:
Inline. I know the code before the change had this as a variable, but it was redundant.